global_vars.tf

```diff
@@ -74,3 +74,8 @@ variable "dns_zone_name" {
 variable "certificate_arn" {
   description = "ARN of the IAM loaded TLS certificate for public ELB"
 }
+
+variable "auto_deploy_updated_tags" {
+  description = "Should updated tags of images be automatically deployed"
```

Comment:
The description should be definite rather than conditional `Automatically deploy images when tags updated`